src/quacc/recipes/common/phonons.py

```diff
@@ -48,12 +52,22 @@ def phonon_subflow(
     """
     Calculate phonon properties.
 
+    In Quacc the ASE constraints can be used to fix atoms. These atoms will
+    not be displaced during the phonon calculation. This will greatly reduce
+    the computational cost of the calculation. However, this is an important
+    approximation and should be used with caution.
+
     Parameters
     ----------
     atoms
         Atoms object with calculator attached.
     force_job
         The static job to calculate the forces.
+    additional_atoms
+        Additional atoms to add to the supercells i.e. fixed atoms.
+        These atoms will not be displaced during the phonon calculation.
+        Useful for adsorbates on surfaces with weak coupling etc.
```

Comment:
I completely see where this confusion can get from, this is why I am not entirely sure about the current name.

Although this forces the users to stop and think before using this important approximation

Reply:
One downside of `fixed_atoms` is that it potentially implies that they are sites within `atoms` (the first positional argument) that are fixed, whereas in reality this is a separate `Atoms` object. That's one reason why I like `additional_atoms` since it makes it clear that this is a separate `Atoms` object, although one would understand that if they read the type hint. Of course, `additional_atoms` doesn't inherently specify anything about them being fixed until you read the docstring.

I think either could be okay, personally.